packages/accounts-password/password_server.js

```diff
@@ -169,18 +169,29 @@ Accounts.registerLoginHandler("password", options => {
 
   check(options, {
     user: Accounts._userQueryValidator,
-    password: passwordValidator
+    password: passwordValidator,
+    token: Match.Optional(NonEmptyString),
   });
 
 
   const user = Accounts._findUserByQuery(options.user, {fields: {
     services: 1,
+    twoFactorAuthentication: 1,
     ...Accounts._checkPasswordUserFields,
   }});
   if (!user) {
     Accounts._handleError("User not found");
   }
 
+  if(Accounts.checkUserHas2FAEnabled && Accounts.checkUserHas2FAEnabled(options.user)){
```

Comment:
add a comment explaining that this method is defined in accounts-2fa package and if it doesn't exist is because this package is not used inside the current app.

Reply:
done